src/qibolab/_core/instruments/qm/program/acquisition.py

```diff
@@ -56,7 +56,7 @@ class Acquisition(ABC):
     @property
     def name(self):
         """Identifier to download results from the instruments."""
-        return f"{self.operation}_{self.element}"
+        return f"{self.operation}_{self.element}".replace("/", "|")
```

Comment:
This took me several hours to find and it probably looks weird, however apparently qm-qua 1.2.1a2 doesn't like the `/` character in stream processing names... I will report this to them.